### PR TITLE
minor edits to markdown, add resources

### PR DESCRIPTION
**Pull request by linda-seiter**
Refactor existing module 2 string format lesson.  Minor edits to markdown and resources.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,259 @@
...
+
+- The **flag** format specifier contains a set of characters to specify the
+  format of the output. The character set very much depends on the data type
+  that is to be formatted.
+- The **width** specifier specifies the amount of characters we want to output
+  when formatting the value.
+- The **precision** specifier is usually used when the data type we are
+  formatting is a `Float` and specifies how many decimal values should be
+  printed.
+- Notice that the flat, width, and precision format specifiers are in brackets.
```
Comment (developerjaym) [inline README.md]: Should 'flat' be 'flag'?

**Review by kcleland0818 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kcleland0818) [reply README.md]: Oops. That typo is from me - sorry

Comment (linda-seiter) [reply README.md]: fixed

**Review by developerjaym — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -252,9 +366,128 @@ Parks and Recreation has 7 seasons.
...
+Potatoes $4.270000
+Eggs $3.990000
+```
+
+### TASK #2
+
+<details>
+    <summary>
+    Update the <code>productFormat</code>
+    format string to format the price with display 2 digits after the decimal point.
```
Comment (kcleland0818) [inline README.md]: "Update the `productFormat` format string to format the price with display 2 digits after the decimal point."

Either remove the "display" or rewrite to: "Update the `productFormat` format string to display only 2 digits after the decimal point."

Comment (linda-seiter) [reply README.md]: fixed

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**